docs/docs/overview/performance.md

```diff
@@ -6,103 +6,202 @@ This section analyzes the performance of Constellation.
+- Machines: `n2d-standard-4`: 2nd Generation AMD EPYC (Rome) processor with 4 Cores, 16 GiB of memory
 - CVM: `true`
-- Region: `West Europe`
+- Zone: `europe-west3-b`
+
+#### AKS
+
+On AKS, we ran the benchmark with Kubernetes `v1.24.9` and nodes with version `AKSUbuntu-1804gen2containerd-2023.02.15`.
+The version we tested on AKS ran with the [`kubenet`](https://learn.microsoft.com/en-us/azure/aks/concepts-network#kubenet-basic-networking) CNI and the [default CSI driver](https://learn.microsoft.com/en-us/azure/aks/azure-disk-csi) for Azure Disk.
+
+We used the following infrastructure configurations.
+
+- Nodes: 2 (2 Worker)
+- Machines: `D4as_v5`: 3rd Generation AMD EPYC 7763v (Milan) processor with 4 Cores, 16 GiB memory
+- CVM: `false`
+- Region: `West US`
 - Zone: `2`
 
-#### K-Bench
+#### GKE
+
+On GKE, we used Kubernetes `v1.24.9` and nodes with version `1.24.9-gke.3200`.
+The version we tested on GKE ran with the [`kubenet`](https://cloud.google.com/kubernetes-engine/docs/concepts/network-overview) CNI and the [default CSI driver](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/gce-pd-csi-driver) for Compute Engine persistent disk.
 
-Using the default [K-Bench test configurations](https://github.com/vmware-tanzu/k-bench/tree/master/config), we ran the following tests on the clusters:
+We used the following infrastructure configurations.
+
+- Nodes: 2 (2 Worker)
+- Machines: `n2d-standard-4` 2nd Generation AMD EPYC (Rome) processor with 4 Cores, 16 GiB of memory
+- CVM: `false`
+- Zone: `europe-west3-b`
 
-- `default`
-- `dp_network_internode`
-- `dp_network_intranode`
-- `dp_fio`
 
 ### Results
 
-#### Kubernetes API Latency
+#### Network
 
-At its core, the Kubernetes API is the way to query and modify a cluster's state. Latency matters here. Hence, it's vital that even with the additional level of security from Constellation's network the API latency doesn't spike.
-K-Bench's `default` test performs calls to the API to create, update, and delete cluster resources.
+We performed a thorough analysis of the network performance of Constellation, specifically focusing on measuring TCP and UDP bandwidth.
+The benchmark measured the bandwidth of pod-to-pod and pod-to-service connections between two different nodes using [`iperf`](https://iperf.fr/).
 
-The three graphs below compare the API latencies (lower is better) in milliseconds for pods, services, and deployments.
+GKE and Constellation on GCP had a maximum network bandwidth of [10 Gbps](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machineshttps://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines).
+Constellation on Azure had a maximum network bandwidth of [1.25 Gbps](https://learn.microsoft.com/en-us/azure/virtual-machines/dcasv5-dcadsv5-series#dcasv5-series-products).
+AKS had a maximum network bandwidth of [12.5 Gbps](https://learn.microsoft.com/en-us/azure/virtual-machines/dasv5-dadsv5-series#dasv5-series).
```

Comment:
Should we make the meaning and consequence of this clearer already here? So people are less negatively surprised if they see the graph?
But I'm not sure if this helps. Many people probably skim the page and just notice the graphs. Can we add a hint inside the graphs that this hits a limitation of the CVMs? Or can we add AKS with CVMs? (Does it have the same limitation?) Or both add AKS with CVMs and the limitation hint?

Reply:
I've discussed with Felix and grouped the graphs by cloud provider for now while running AKS with CVMs.